enso/resample/__init__.py

```diff
@@ -0,0 +1,14 @@
+from imblearn.over_sampling import SMOTE 
+from imblearn.over_sampling import ADASYN
+
+
+def resample(resample_type, train_data, train_labels):
+    if resample_type.lower() == 'none':
+        return train_data, train_labels
+    else:
+        instance = globals()[resample_type]()
```

Comment:
Rather than the globals call and the functional layout here I think I would just upgrade the resampler to a straight up additional step in the pipeline. Basically refactoring this to be a base class for all resamplers and then making smote and adasyn specific classes (though you could set it up so that they're generated in a one-liner)

Main reason is that I think resampling is another thing we'd like to experiment with and figuring out how big the impact is, so this'll just make things a little bit less scalable. SMOTE and ADASYN are well structured, but a lot of the other sampling mechanisms in imblearn don't fit the same method signature.